data/hai/hai reveal 5 philosophers.txt

```diff
@@ -732,37 +767,97 @@ mission "Hai Reveal [B11] Moving Together"
-			`	"Now before we get carried away, we have some introductions to do. I don't think everyone here knows everyone yet."`
+			label briefing
+			choice
+				`	"I see new faces here. Can everyone introduce themselves?"`
+					goto introductions
+					to display
+						not "Hai Reveal [B11] Moving Together: label introductions"
+				`	"What have you found about the smuggling routes?"`
+					goto smuggling
+					to display
+						not "Hai Reveal [B11] Moving Together: label smuggling"
+				`	"How have you dealt with the smugglers so far?"`
+					goto dealt
+					to display
+						has "Hai Reveal [B11] Moving Together: label smuggling"
+						not "Hai Reveal [B11] Moving Together: label dealt"
+				`	"How do the Hai feel about the approach to the smugglers?"`
+					goto hai
+					to display
+						has "Hai Reveal [B11] Moving Together: label smuggling"
+						not "Hai Reveal [B11] Moving Together: label hai"
+				`	"Are you still able to keep the Hai a secret?"`
+					goto secret
+					to display
+						not "Hai Reveal [B11] Moving Together: label secret"
+				`	"What do you know about Elenchus?"`
+					goto elenchus
+					to display
+						not "Hai Reveal [B11] Moving Together: label elenchus"
+				`	"How will we penetrate the Elenchus defenses?"`
+					goto penetration
+					to display
+						has "Hai Reveal [B11] Moving Together: label elenchus"
+						not "Hai Reveal [B11] Moving Together: label penetration"
+				`	"We've discussed enough. It's time to get started."`
+					goto solution
+			label smuggling
+			action
+				set "Hai Reveal [B11] Moving Together: label smuggling"
+			`	Remington looks at you. "I know its rough when news travels ahead of you, but Moray's team identified two bases which act as holding facilities in human space for the previously unidentified group that you've been investigating. That's where it seems the largest number of kidnapped Hai have been taken for processing before being taken to this mysterious 'Elenchus' place you've found. One of the bases is on Deadman's Cove, and arrangements are already being made to deal with it, but the other one is right here on Stormhold. In fact it narrowed down our remaining search here to a single target in one area up in the north. It should take us no more than a few days to pinpoint it, and a few days more to plan and execute an action against it." He shrugs. "The information was just too important to delay it for your arrival. That, and they decided they wanted me here personally since Terry appears to be busy off doing something else."`
```

Comment:
```suggestion
			`	Remington looks at you. "I know it's rough when news travels ahead of you, but Moray's team identified two bases which act as holding facilities in human space for the previously unidentified group that you've been investigating. That's where the largest number of kidnapped Hai have been taken for processing, before being taken to this mysterious 'Elenchus' place you've found. One of the bases is on Deadman's Cove, and arrangements are already being made to deal with it, but the other one is right here on Stormhold. In fact, it narrowed down our remaining search here to a single target in one area up in the north. It should take us no more than a few days to pinpoint it, and a few days more to plan and execute an action against it." He shrugs. "The information was just too important to delay it for your arrival. That, and they decided they wanted me here personally since Terry appears to be busy off doing something else."`
```